### PR TITLE
chore: update user management page.

### LINKED ISSUE #3484: [Content Bug] Update Members list for enterprise customers.
I was troubleshooting a case for an enterprise customer, and the Biz ops team confirmed that user management for enterprise customers is different.
Here's more info:
There shouldn't be an invite accept link in the email - Postman has mandatory SAML, so it should say "Use your company identity provider to log in to Timescale" instead of a button to accept the invite. The only way for mandatory SAML users to accept invites is through the notification bell in the console.
Is it possible to add this in [the docs](https://docs.timescale.com/use-timescale/latest/members/members-list/) for reference? (edited)

### PR DESCRIPTION
**Pull request by billy-the-fish**
Page to review is: https://docs-dev.timescale.com/docs-3484-content-bug-update-members-list-for-enterprise-customers/use-timescale/3484-content-bug-update-members-list-for-enterprise-customers/members/.

### CODE REVIEW

File: use-timescale/members.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+keywords: [members, projects, admin, roles]
+tags: [users]
+cloud_ui:
+    path:
+        - [members]
+---
+
+# User management
+
+When you sign up for a [30 day free trial][sign-up], $CLOUD_LONG creates a project for you, and
```
Comment (atovpeko) [inline use-timescale/members.md]:
```suggestion
When you sign up for a [30-day free trial][sign-up], $CLOUD_LONG creates a project for you, and
```

File: use-timescale/members.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+2. Select the project you want to use.
+
+You can now manage the users and services in this project
+
+</Procedure>
+
+## Transfer project ownership
+
+Each project in $CONSOLE has one `owner`. As the project owner, you have rights to
+to add and delete users, edit project settings and transfer the owner role to another user. When you transfer
```
Comment (atovpeko) [inline use-timescale/members.md]:
```suggestion
add and delete users, edit project settings, and transfer the owner role to another user. When you transfer
```

File: use-timescale/members.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+
+## Remove users from a project
+
+To remove a user's access to a project:
+
+<Procedure>
+
+1.  In [$CONSOLE][cloud-login], click `Invite users`.
+1.  Next to the person you want to remove, click `⋮`, then click the trash can.
```
Comment (atovpeko) [inline use-timescale/members.md]:
```suggestion
1.  Next to the person you want to remove, click `⋮`, then click the trash can icon.
```

**Review by atovpeko — CHANGES_REQUESTED** [review]
Can we align to only use bold for UI elements, not code font? To quote the style guide: When referring to any UI element by name, put its name in bold, using the b element in HTML or ** in Markdown. This includes names for buttons, menus, dialogs, windows, list items, or any other feature on the page that has a visible name. Don't use code font for UI elements, unless it's an element that meets the [requirements for code font](https://developers.google.com/style/code-in-text). In that case, use both code font and bold

**Review by mwarejohn — APPROVED** [review]
Enterprise user section is okay

File: use-timescale/members.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+- **Existing $CLOUD_LONG user**:
+  1. In the invitation email, click **Accept Invite**.
+
+     $CLOUD_LONG opens and you are added to the project.
+
+- **Enterprise $CLOUD_LONG user**:
+  1. In the invitation email, click **Use your company identity provider to log in to Timescale**.
+
+     $CLOUD_LONG opens and you are added to the project.
```
Comment (syvb) [inline use-timescale/members.md]: SAML users aren't automatically added to the project; they need to accept the invite from the notification bell.

Comment (billy-the-fish) [reply use-timescale/members.md]: Hi @syvb , thanks for the comment. I'm afraid I don't know what the notification bell is. Can you tell me what the workflow for SAML users is please (Do they have to be enterprise users?). For example:

1. User is invited to a project by the Owner or another project member, then: 
    - Existing user: do this
    - New user: do that

Comment (syvb) [reply use-timescale/members.md]: This is the notification bell:
image

Normally customers have to be on the Enterprise tier for us to enable SAML for them, but we can make exceptions if needed. How the flow works for SAML users is:

1. Invitee is invited to the project by another project member
2. Invitee logs in to the Timescale console through their company's identity provider.
3. Invitee clicks the notification bell in the bottom left (which used to be in the top right before the recent console redesign)
4. Invitee clicks the button to accept the invite
5. Invitee uses the project switcher in the top left to switch between the project they were just invited to and their default project

Comment (billy-the-fish) [reply use-timescale/members.md]: Thanks for the guidance, that really helped. I have updated the docs.

**Review by syvb — APPROVED** [review]

**Review by atovpeko — APPROVED** [review]

**Pull request merged**